src/tests/JIT/Directed/Convert/out_of_range_fp_to_int_conversions.cpp

```diff
@@ -18,6 +18,7 @@ typedef enum {
     CONVERT_SATURATING,
     CONVERT_NATIVECOMPILERBEHAVIOR,
     CONVERT_MANAGED_BACKWARD_COMPATIBLE,
```

Comment:
Rename to `CONVERT_MANAGED_BACKWARD_COMPATIBLE_X86_X64` to match the managed def?